lib/presentation/widget/upload_file/upload_file_widget.dart

```diff
@@ -258,6 +266,13 @@ class _UploadFileWidgetState extends State<UploadFileWidget> {
     );
   }
 
+  String displayFileName(String filename)
+  {
+    final sublength = filename.length - filename.split('.').last.length;
```

Comment:
let's follow `lowerCamelCase` for variables: https://dart.dev/guides/language/effective-dart/style